hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -159,6 +159,7 @@
 
     SCHEMA_CACHE_REFRESH_AFTER_WRITE_MINUTES("schema.cache.refresh.after.write.minutes", 10),
     SCHEMA_CACHE_EXPIRE_AFTER_WRITE_MINUTES("schema.cache.expire.after.write.minutes", 60 * 24),
+    SCHEMA_CACHE_EXPIRE_AFTER_ACCESS_MINUTES("schema.cache.expire.after.access.minutes", 60 * 48),
```

Comment:
I'd say it should be `SCHEMA_CACHE_COMPILED_EXPIRE_AFTER_ACCESS_MINUTES` (near existing `SCHEMA_CACHE_COMPILED_MAXIMUM_SIZE `), because it's a different cache than the one configured by upper properties.